cmd/validator/validator.go

```diff
@@ -72,32 +84,60 @@ func validatorUsage() {
 // will return with exit = 1
 func getFlags() (validatorConfig, error) {
 	flag.Usage = validatorUsage
+	reporterConfigFlags := reporterFlags{}
+
 	depthPtr := flag.Int("depth", 0, "Depth of recursion for the provided search paths. Set depth to 0 to disable recursive path traversal")
 	excludeDirsPtr := flag.String("exclude-dirs", "", "Subdirectories to exclude when searching for configuration files")
 	excludeFileTypesPtr := flag.String("exclude-file-types", "", "A comma separated list of file types to ignore")
-	outputPtr := flag.String("output", "", "Destination to a file to output results")
-	reportTypePtr := flag.String("reporter", "standard", "Format of the printed report. Options are standard, json, junit and sarif")
 	versionPtr := flag.Bool("version", false, "Version prints the release version of validator")
 	groupOutputPtr := flag.String("groupby", "", "Group output by filetype, directory, pass-fail. Supported for Standard and JSON reports")
 	quietPtr := flag.Bool("quiet", false, "If quiet flag is set. It doesn't print any output to stdout.")
+	flag.Var(
+		&reporterConfigFlags,
+		"reporter",
+		`A colon-separated string of report formats with optional output file paths.
```

Comment:
The format are not separated by colon,unless I'm wrong.

The filename is after the colon, but it's optional.

It requires a rewriting

Reply:
yeah, i basically meant here that colon separated string with format and optional file path. Do you feel like it needs a rewrite?

Reply:
Thanks for updating. It's clearer now